test/setup.sh

```diff
@@ -13,11 +13,11 @@ echo Copying weave images and script to hosts
 for HOST in $HOSTS; do
     docker_on $HOST load -i ../weave.tar
     docker_on $HOST load -i ../weavedns.tar
-    docker_on $HOST load -i ../weavetools.tar
-    run_on $HOST mkdir -p `dirname $WEAVE`
-    cat ../weave         | run_on $HOST sh -c "cat > $WEAVE"
+    docker_on $HOST load -i ../weaveexec.tar
+    cp ../weave ./
```

Comment:
Oh. Probably because I wanted to invoke weave from the current directory; but `WEAVE=../weave` after all, so it's not needed.